src/module/script.js

```diff
@@ -0,0 +1,44 @@
+import './index.js';
+
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const checkbox = document.querySelector('#myTasksList, #checkbox');
+
+const saveTasks = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+function checkCheckbox() {
+  tasks.forEach((task) => {
+    if (checkbox.checked) {
+      task.completed = false;
+    } else {
+      task.completed = true;
+    }
+  });
+  saveTasks();
+}
+
+checkbox.addEventListener('change', checkCheckbox);
+
+const updateTaskIndexes = () => {
+  tasks.forEach((task, index) => {
+    task.index = index;
+  });
+};
+const clearCompletedTasks = () => {
+  const checkboxes = document.querySelectorAll('input[type="checkbox"]');
+  checkboxes.forEach((checkbox) => {
+    if (checkbox.checked) {
+      const taskContainer = checkbox.closest('.content');
+      taskContainer.remove();
+      const taskId = taskContainer.getAttribute('index');
+      tasks = tasks.filter((task) => task.index !== taskId);
+    }
+  });
+  saveTasks();
+  updateTaskIndexes();
+};
+
+const clearAll = document.getElementById('remove');
+clearAll.addEventListener('click', clearCompletedTasks);
```

Comment:
- [ ] Hi, great work on the todo list. I noticed that toggling of tasks between complete and not completed to the todo list are not being persisted correctly to local storage. When I mark a todo item as complete, all items are deleted from the local storage. You might want to check your logic for updating the local storage when a todo item is completed. You should only remove the completed items  that has been cleared when the `clear all completed` button is clicked from the local storage, not all items. This will improve the functionality and user experience of your app.